README.md

```diff
@@ -226,6 +256,22 @@ You can use the following functionalities when interacting with USS files:
 ![Refresh All](docs/images/ZE-refreshUSS.gif?raw=true "Refresh All")
 <br /><br />
 
+#### Rename USS files
+
+1. Navigate to the explorer tree.
+2. Open the **USS** bar.
+3. Select a USS file you want to rename.
+4. Right-click the USS file and select the **Rename USS file** option.
+5. Change the name of the USS file.
+
+#### Copy USS files
```

Comment:
Copy functionality for USS files has not yet been implemented for this next release